### PR TITLE
Upgrading to google-java-format 1.1

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**
Also reformatting everything.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -14,7 +14,7 @@ buildscript {
   }
   dependencies {
     classpath "com.google.protobuf:protobuf-gradle-plugin:0.7.3"
-    classpath "gradle.plugin.com.github.sherter.google-java-format:google-java-format-gradle-plugin:0.2"
+    classpath "gradle.plugin.com.github.sherter.google-java-format:google-java-format-gradle-plugin:0.3.2"
```
Comment (pongad) [inline build.gradle]: I don't think this actually changes the version of the formatter we use to format our code.

We use formatter in two places. One to format and verify Java code in toolkit itself and another to format the java files we render. If we look at the [build log](https://travis-ci.org/googleapis/toolkit/jobs/164732499#L661), we can see that the format verification is still done with version 1.0. I have no idea why the formatting in the comment changed. It's possible that the gradle plugin adds a new option to the formatter.

To change the version of the formatter we use to format toolkit, I think we need to change [this line](https://github.com/garrettjonesgoogle/toolkit/blob/7a75b5aa164d18026de8acc08ebf4366b0598ff5/build.gradle#L446)

File: build.gradle [hunk 2]
```diff
@@ -73,7 +73,7 @@ ext {
     jacksonGuava: 'com.fasterxml.jackson.datatype:jackson-datatype-guava:2.7.0',
 
     // Formatter
-    javaFomatter: 'com.google.googlejavaformat:google-java-format:0.1-alpha'
+    javaFomatter: 'com.google.googlejavaformat:google-java-format:1.1'
```
Comment (pongad) [inline build.gradle]: I think this only changes the formatter we format our rendered code with.

File: build.gradle (hunk 1 above)
Comment (garrettjonesgoogle) [inline build.gradle]: Ahh, I didn't realize the two were separate.

**Comment by garrettjonesgoogle** [comment]
The main thing I wanted to update was the version that I format the GAPIC output with. I have undone the  change to the version of google-java-format-gradle-plugin and consequently all of the changes to toolkit code, so now this is a pretty trivial PR.

**Comment by garrettjonesgoogle** [comment]
(Note - I recognize that I didn't follow my own advice on the PR reset here, but it was going to be more difficult than it was worth to reverse all of the formatting changes)

**Comment by garrettjonesgoogle** [comment]
(PTAL)

**Comment by pongad** [comment]
@garrettjonesgoogle Do we want to change the version of the formatter we format toolkit with eventually?

**Comment by garrettjonesgoogle** [comment]
@pongad yes, but I found that the gradle plugin doesn't work with the latest google-java-format (a class has been removed that the plugin references). We'll need to wait until the plugin is updated.

**Pull request merged**